unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayerNative.cs

```diff
@@ -0,0 +1,73 @@
+{
+    public class WebVideoPlayerNative : IWebVideoPlayerPlugin
+    {
+#if UNITY_WEBGL && !UNITY_EDITOR
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerCreate(string id, string url, bool useHls);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerRemove(string id);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerTextureUpdate(string id, IntPtr texturePtr, bool isWebGL1);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerPlay(string id, float startTime);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerPause(string id);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerVolume(string id, float volume);
+    [DllImport("__Internal")]
+    private static extern int WebVideoPlayerGetHeight(string id);
+    [DllImport("__Internal")]
+    private static extern int WebVideoPlayerGetWidth(string id);
+    [DllImport("__Internal")]
+    private static extern float WebVideoPlayerGetTime(string id);
+    [DllImport("__Internal")]
+    private static extern float WebVideoPlayerGetDuration(string id);
+    [DllImport("__Internal")]
+    private static extern int WebVideoPlayerGetState(string id);
+    [DllImport("__Internal")]
+    private static extern string WebVideoPlayerGetError(string id);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerSetTime(string id, float second);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerSetPlaybackRate(string id, float playbackRate);
+    [DllImport("__Internal")]
+    private static extern void WebVideoPlayerSetLoop(string id, bool loop);
+#else
+        private static void WebVideoPlayerCreate(string id, string url, bool useHls) { }
+        private static void WebVideoPlayerRemove(string id) { }
+        private static void WebVideoPlayerTextureUpdate(string id, IntPtr texturePtr, bool isWebGL1) { }
+        private static void WebVideoPlayerPlay(string id, float startTime) { }
+        private static void WebVideoPlayerPause(string id) { }
+        private static void WebVideoPlayerVolume(string id, float volume) { }
+        private static int WebVideoPlayerGetHeight(string id) { return 0; }
+        private static int WebVideoPlayerGetWidth(string id) { return 0; }
+        private static float WebVideoPlayerGetTime(string id) { return 0; }
+        private static float WebVideoPlayerGetDuration(string id) { return 0; }
+        private static int WebVideoPlayerGetState(string id) { return (int)VideoState.ERROR; }
+        private static string WebVideoPlayerGetError(string id) { return "WebVideoPlayer: Platform not supported"; }
+        private static void WebVideoPlayerSetTime(string id, float second) { }
+        private static void WebVideoPlayerSetPlaybackRate(string id, float playbackRate) { }
+        private static void WebVideoPlayerSetLoop(string id, bool loop) { }
+#endif
```

Comment:
Cool refactor. We can't move the static methods to a `Utils` class or something similar? This code would be far easier to follow with the change.

Reply:
Wouldn't the Utils class be the same as this then? you need to handle the existence of this DLL on multiple platforms and this class just does that